plugin.py

```diff
@@ -31,7 +31,9 @@ def plugin_info(self):
         return plugin_info
 
     def process(self, trace, data_context):
-        kaitai_utils.write_kaitai_to_trace(trace)
+        bytearray_length = 64
```

Comment:
Is it possible to make this configurable? Or did we decide on 64 as a hard limit?

Reply:
Also, I think it should be `byte_array_length`:
```suggestion
        byte_array_length = 64
```